code/modules/mob/living/carbon/alien/humanoid/caste/hunter.dm

```diff
@@ -49,8 +52,12 @@
 		src << "<span class='alertalien'>It is unsafe to leap without gravity!</span>"
 		//It's also extremely buggy visually, so it's balance+bugfix
 		return
-
-	else //Maybe uses plasma in the future, although that wouldn't make any sense...
+	if(user.getPlasma() < plasma_cost)
+		if(!silent)
+			user << "<span class='noticealien'>Not enough plasma stored.</span>"
+		return 0
+	else // Added plasma cost for balance
+		user.adjustPlasma(-15)

```

Comment:
Use `user.usePlasma()` it has the added bonus returning a success or fail to eliminate the need for a plasma level check.